embedded-graphics/src/draw_target.rs

```diff
@@ -100,8 +107,18 @@ use crate::{
 ///
 /// In addition to defining [`draw_pixel`], an implementation of [`DrawTarget`] can also provide
 /// alternative implementations for hardware accelerated drawing operations. This example implements
-/// `DrawTarget` for a display that supports drawing hardware accelerated styled [`Rectangle`]s. The
-/// `draw_rectangle()` method overrides the default implementation.
+/// `DrawTarget` for a display without a framebuffer that supports hardware accelerated drawing of
+/// styled [`Rectangle`]s.
+///
+/// The default implementation of `draw_rectangle()` (as well as `draw_line()`, `draw_circle()` and
+/// `draw_triangle()`) defer to `draw_iter()` internally. In this example, the default
+/// implementation of `draw_rectangle()` is overridden to allow usage of accelerated draw commands
+/// specific to the targeted hardware.
```

Comment:
IMO the function references should be links and the `()` removed.
Explicitly listing all `draw_...` functions increases the risk of this comment to become out of sync with the code, when additional methods are added.

Reply:
Good shout. I've tweaked it a bit. I think the intent is pretty clear now so let's leave this doc section as is.